src/app/artist-details/artist-details.component.css

```diff
@@ -0,0 +1,103 @@
+/* You can add global styles to this file, and also import other style files */
```

Comment:
Please remove all comments from this file.

Reply:
@strizzwald fixed